src/MySqlConnector/Core/CommandListPosition.cs

```diff
@@ -17,6 +18,11 @@ public CommandListPosition(IReadOnlyList<IMySqlCommand> commands)
 	/// </summary>
 	public IReadOnlyList<IMySqlCommand> Commands { get; }
 
+	/// <summary>
+	/// Associated prepared statements of commands
+	/// </summary>
+	public PreparedStatements? PreparedStatements;
```

Comment:
Note to self: this struct is starting to accumulate a lot of mutable public fields; redesign?